src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -15,11 +15,11 @@ public sealed class DiagnosticsTelemetryModule : ITelemetryModule, IDisposable
 
         internal readonly DiagnosticsListener EventListener;
 
+        internal IHeartbeatProvider HeartbeatProvider = null;
```

Comment:
mark readonly so you can safely assume it's not `null`

Reply:
Done